tests/models/test_models.py

```diff
@@ -20,6 +20,7 @@
     "stabilityai/stablelm-3b-4e1t",
     "allenai/OLMo-1B",
     "bigcode/starcoder2-3b",
+    "core42/jais-13b",
```

Comment:
Currently CI machine might not support >7b models and we can remove this test.

Reply:
This is done.